kfs-ld/src/main/java/edu/arizona/kfs/module/ld/batch/service/impl/ScrubberValidatorImpl.java

```diff
@@ -88,37 +87,37 @@
             errors.add(err);
         }
 
-        err = validateGTEAccount(laborOriginEntry, laborScrubbedEntry, (LaborAccountingCycleCachingService) laborAccountingCycleCachingService);
+        err = validateGTEAccount(laborScrubbedEntry);
         if (err != null) {
             errors.add(err);
         }
 
         return errors;
     }
 
-    protected Message validateGTEAccount(LaborOriginEntry laborOriginEntry, LaborOriginEntry laborScrubbedEntry, LaborAccountingCycleCachingService laborAccountingCycleCachingService) {
+    protected Message validateGTEAccount(LaborOriginEntry originEntry) {
         LOG.debug("validateGTEAccount() started");
-        laborOriginEntry.refreshReferenceObject("account");
-        laborOriginEntry.refreshReferenceObject("financialObject");
-        Account account = laborOriginEntry.getAccount();
+        originEntry.refreshReferenceObject("account");
```

Comment:
refreshReferenceObject in a validate method: are we sure that by this point we don't have the complete object yet?